### PR TITLE
Fix a couple of funny uses of \si.

### PR DESCRIPTION
**Pull request by bangerth**
Mentioned in https://github.com/geodynamics/aspect/pull/3798#discussion_r479693148

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -931,8 +930,8 @@ \subsection{Coefficients}
 \item \textit{The change in entropy $\Delta S$ at a
   phase transition together with the derivatives of the phase function
   $X=X(p,T,\mathfrak c,\mathbf x)$ with regard to temperature and pressure:} Units
-  $\frac{\textrm{J}}{\si{kg}\si{K}^2}$ ($-\Delta S \frac{\partial X}{\partial T}$) and
-  $\frac{\si{m}^3}{\si{kg}\si{K}}$ ($\Delta S \frac{\partial X}{\partial p}$).
+  \si{K/kg/K^2 ($-\Delta S \frac{\partial X}{\partial T}) and
```
Comment (bobmyhill) [inline doc/manual/manual.tex:933]:
```suggestion
  \si{J/kg/K^2 ($-\Delta S \frac{\partial X}{\partial T}) and
```

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -811,16 +811,15 @@ \subsection{Coefficients}
...
 
   The specific heat capacity denotes the amount of energy needed to increase
   the temperature of one kilogram of material by one Kelvin at constant pressure.
   Wikipedia lists a value of
-  790 $\frac{\textrm{J}}{\si{kg}\cdot\si{K}}$ for granite%
+  $790 \si{J/kg/K}$ for granite%
   \footnote{See \url{http://en.wikipedia.org/wiki/Specific_heat}.}
-  For the earth mantle, a value of 1250
-  $\frac{\textrm{J}}{\si{kg}\cdot\si{K}}$ is within the range
+  For the earth mantle, a value of $1250
```
Comment (bobmyhill) [inline doc/manual/manual.tex:821]:
```suggestion
  For the Earth's mantle, a value of $1250
```

**Review by bobmyhill — CHANGES_REQUESTED** [review]

File: doc/manual/manual.tex [hunk 3]
```diff
@@ -931,8 +930,8 @@ \subsection{Coefficients}
 \item \textit{The change in entropy $\Delta S$ at a
   phase transition together with the derivatives of the phase function
   $X=X(p,T,\mathfrak c,\mathbf x)$ with regard to temperature and pressure:} Units
-  $\frac{\textrm{J}}{\si{kg}\si{K}^2}$ ($-\Delta S \frac{\partial X}{\partial T}$) and
-  $\frac{\si{m}^3}{\si{kg}\si{K}}$ ($\Delta S \frac{\partial X}{\partial p}$).
+  \si{K/kg/K^2 ($-\Delta S \frac{\partial X}{\partial T}) and
+  \si{m^3/kg/K} ($\Delta S \frac{\partial X}{\partial p}).
```
Comment (bobmyhill) [inline doc/manual/manual.tex:934]:
```suggestion
  \si{m^3/kg/K} ($\Delta S \frac{\partial X}{\partial p}$).
```

File: doc/manual/manual.tex (hunk 1 above)
Comment (bangerth) [reply doc/manual/manual.tex:933]: Oh, 3 mistakes in one line...

**Comment by bangerth** [comment]
Thanks, @bobmyhill . So fixed!

**Comment by bobmyhill** [comment]
> Thanks, @bobmyhill . So fixed!

Great!

**Review by gassmoeller — APPROVED** [review]
Looks good to me. @bobmyhill do you think this is ready to merge?

**Comment by bobmyhill** [comment]
@gassmoeller, yes, this looks good to me!

**Review by bobmyhill — APPROVED** [review]

**Pull request merged**